src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -89,8 +89,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
         c.argument('no_wait', help="Do not wait for the run to complete and return immediately after queuing the run.", action='store_true')
         c.argument('no_format', help="Indicates whether the logs should be displayed in raw format", action='store_true')
-        c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', arg_type=get_enum_type(OsType))
-        c.argument('platform', help='The platform of the resulting image.')
+        c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', arg_type=get_enum_type(OsType), deprecate_info=c.deprecate(hide=True))
+        c.argument('platform', help="The platform of the resulting image in the format of 'os/arch/variant'. This replaces the '--os' flag which is soon to be deprecated. E.g. linux/arm/v7. The 'arch' and 'variant' parts are optional.")
```

Comment:
For deprecate_info, add a redirect to platform. Don't describe the replacement in your help text.  https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments